### DISCUSSION TITLE
a soldier with animation for starter script

### DISCUSSION THREAD

**Discussion by blotterchains** [Godot-ground/tower-defense#3] [Announcements]
hello godoers,

I have been working on the core of the game for the past few days and the code part of the game is almost finished. Please make a soldier with an arrow from the artist section. I would also like to say that the soldier you design should only separate its different parts.  For example, the soldier should include head, arm, elbow, hand, body, thigh, leg, knee, foot and each of them should be a separate file so that the animator can create animation much more easily.  Dont make traditional design poses This takes a lot of time and we have less freedom in designing animations and body physics.

**Comment by Alinj7** [comment]
The following games can be a good model for soldier design:

1. Metal Soldiers
2. Mini Militia
3. Guns up
4. World war defense
5. Clone Armies
